include/wx/qt/mdi.h

```diff
@@ -29,15 +34,54 @@ class WXDLLIMPEXP_CORE wxMDIParentFrame : public wxMDIParentFrameBase
                 long style = wxDEFAULT_FRAME_STYLE | wxVSCROLL | wxHSCROLL,
                 const wxString& name = wxASCII_STR(wxFrameNameStr));
 
+    QMdiArea* GetQtMdiArea() const;
+
+    void QtSetPreferredDI(bool isTDI = true);
```

Comment:
Are we sure we're never going to have other variants? I'd use an enum here, e.g. something like

```cpp
enum class Layout
{
  Tabbed,
  MDI
};
```

Reply:
I made this change with `MDI` first to be compatible with `QMdiArea::ViewMode` `enum`. And change `QtSetPreferredDI()` to `QtSetPreferredDILayout()`